b2/console_tool.py

```diff
@@ -1447,6 +1447,8 @@ def get_parser(cls):
     def run_command(self, argv):
         signal.signal(signal.SIGINT, keyboard_interrupt_handler)
         parser = self.get_parser()
+        if len(argv) >= 2:
+            argv[1] = argv[1].replace('_', '-')
```

Comment:
In python3 we could use aliases. Maybe add a comment to use it if we drop support for python2

Reply:
Another way is to monkypatch argparse for python2 and add a functionality for aliases which is pretty straightforward

Reply:
I think this makes `b2 __help` work. This monkeypatching seems like a better idea tbh. @franciscouzo will you try it? Talk to @mlech-reef if you need guidance

Reply:
@franciscouzo Currently I'm working on other improvements and fixes like more descriptive `--help`, better handling argument errors, etc. I had to subclass `argparse.ArgumentParser` anyway in `arg_parser.py:ArgumentParser`. You can do the same, so we will have less merge conflicts.

Reply:
An also please sync your code with `Backblaze:master`. CI has been fixed and now AppVeyor and Travis should be green before the merge.

Reply:
Please see #629

Reply:
@mlech-reef was command with underscores a part of #629 or does it still need some work? Will you take care of it?

Reply:
@ppolewicz #635
Please close this one